inst/components/scss/value_box.scss

```diff
@@ -1,8 +1,104 @@
+$bslib-value-box-enable-border: "auto" !default; // "auto" | "never" | "always"
+$bslib-value-box-enable-shadow: $enable-shadows !default;
+$bslib-value-box-horizontal-min-width: 300px;
+
+// CSS variables that start with --- (3 dashes instead of 2) are private vars
+// that are set by user arguments in the R calls. The CSS properties below only
+// apply to the default value box style; the standard way to theme individual
+// value boxes is with CSS classes on the value box container.
+// :root {
+//   --bslib-value-box-color:;
+//   --bslib-value-box-bg:;
+//   --bslib-value-box-border-color:;
+//   --bslib-value-box-border-width:;
+// }
+
 .bslib-value-box {
-  --bslib-value-box-separator-color: rgba(var(--bs-white-rgb, 255,255,255), 0.175);
+  @if ($bslib-value-box-enable-shadow) {
+    box-shadow: $box-shadow;
+  }
+
+  &.default {
+    --bslib-value-box-bg-default: var(--bs-card-bg, #{$body-bg});
+    --bslib-value-box-border-color-default: var(--bs-card-border-color, #{$card-border-color});
+    --bslib-value-box-border-width-default: var(--bs-card-border-width, #{$card-border-width});
+    color: var(--bslib-value-box-color);
+    background-color: var(--bslib-value-box-bg, var(--bslib-value-box-bg-default));
+    border-color: var(--bslib-value-box-border-color, var(--bslib-value-box-border-color-default));
+    border-width: var(--bslib-value-box-border-width, var(--bslib-value-box-border-width-default));
+  }
+
+  container-name: bslib-value-box;
+  container-type: inline-size;
 
   .value-box-grid {
-    grid-template-columns: var(--bslib-value-box-widths);
+    display: grid;
+    grid-template-areas: "left right";
+    align-items: center;
+  }
+
+  .value-box-showcase {
+    height: 100%;
+    max-height: var(---bslib-value-box-showcase-max-h, 100%);
+  }
+
+  &[data-full-screen="true"] {
+    .value-box-showcase {
+      max-height: var(---bslib-value-box-showcase-max-h-fs, 100%);
+    }
+  }
+
+  @media screen and (min-width: breakpoint-max(sm, $grid-breakpoints)) {
```

Comment:
Probably worth a comment explaining this in high level terms (and calling out why we can't use `@include media-breakpoint` here)

Reply:
```suggestion
  // When value boxes get too small, we lay them out vertically using a container query.
  // But we don't need this for mobile, where we have one box per row. This media query
  // is the inverse of `@include media-breakpoint-down(sm)` but that mixin doesn't work
  // inside nested queries.
  @media screen and (min-width: breakpoint-max(sm, $grid-breakpoints)) {
```